src/setupTests.ts

```diff
@@ -1,4 +1,22 @@
 import { configure } from 'enzyme'
 import Adapter from 'enzyme-adapter-react-16'
+import { JSDOM } from 'jsdom'
 
 configure({ adapter: new Adapter() })
+
+const { window } = new JSDOM('<!doctype html><html><body></body></html>')
+
+const globalAny: any = global
+
+globalAny.window = window
+globalAny.document = window.document
+globalAny.navigator = {
+  userAgent: 'node.js'
+}
+globalAny.requestAnimationFrame = (callback: () => void) => setTimeout(callback, 0)
```

Comment:
Let's use [`raf`](https://github.com/chrisdickinson/raf) instead?
```suggestion
import 'raf/polyfill'
```

Reply:
How is that better than this one line solution?

Reply:
[`requestAnimationFrame`](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame) provides a monotonic [`DOMHighResTimeStamp`](https://developer.mozilla.org/en-US/docs/Web/API/DOMHighResTimeStamp) to its callback, so this _two_ line solution is inaccurate. I'd rather rely on a stable, popular dependency than roll our own if possible.

Reply:
What's inaccurate about it, the typing? I added that myself. The rest of it I copied directly from the enzyme docs
https://airbnb.io/enzyme/docs/guides/jsdom.html

I also strongly favor adding one or two (is it two because of `cancelAnimationFrame` below?) lines to our source vs add yet another dependency to this project.

Reply:
It's inaccurate because it does not call its callback with a monotonic timestamp.

Reply:
[_Using enzyme with JSDOM_](https://airbnb.io/enzyme/docs/guides/jsdom.html) does not provide a valid reason for a ryo polyfill. One would assume that they wanted to keep the reader focused on `jsdom` and keep them from wondering what the heck a `raf` is. I believe we should incorporate this article's message, not blindly copy and paste its content.